<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerfilDialog</class>
 <widget class="QDialog" name="PerfilDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>üë§ Meu Perfil</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog {
    background-color: #1A161A;
}

QScrollArea {
    background-color: #1A161A;
    border: none;
}

QWidget#scrollAreaWidgetContents {
    background-color: #1A161A;
}

QLabel {
    color: #F4B315;
}

QLabel#tituloLabel {
    font-size: 28px;
    font-weight: bold;
    padding: 15px;
}

QLabel#subtituloLabel {
    font-size: 14px;
    color: #8E6915;
}

QFrame {
    background-color: #423738;
    border-radius: 10px;
    padding: 20px;
}

QLineEdit, QSpinBox {
    background-color: #2A2426;
    border: 1px solid #423738;
    color: #F4B315;
    padding: 8px;
    border-radius: 5px;
    font-size: 13px;
}

QLineEdit:focus, QSpinBox:focus {
    border: 2px solid #F4B315;
}

QPushButton {
    background-color: #F4B315;
    color: #1A161A;
    border: none;
    border-radius: 8px;
    padding: 12px 24px;
    font-weight: bold;
    font-size: 13px;
}

QPushButton:hover {
    background-color: #D3AF35;
}

QPushButton#cancelarButton {
    background-color: #4A2020;
    color: #FF6B6B;
}

QPushButton#cancelarButton:hover {
    background-color: #5A2828;
}

QPushButton#alterarSenhaButton {
    background-color: #423738;
    color: #F4B315;
    border: 2px solid #F4B315;
}

QPushButton#alterarSenhaButton:hover {
    background-color: #524447;
}

QScrollBar:vertical {
    background: #2A2426;
    width: 12px;
    border-radius: 6px;
}

QScrollBar::handle:vertical {
    background: #F4B315;
    border-radius: 6px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: #D3AF35;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>996</width>
        <height>972</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="tituloLabel">
         <property name="text">
          <string>üë§ Meu Perfil</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">background-color: #F4B315;</string>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_principal">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_foto">
           <item>
            <widget class="QLabel" name="fotoLabel">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">
background-color: #423738;
border-radius: 75px;
font-size: 80px;
              </string>
             </property>
             <property name="text">
              <string>üë§</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="escolherFotoButton">
             <property name="text">
              <string>üì∑ Alterar Foto</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="usernameLabel">
             <property name="styleSheet">
              <string notr="true">
font-size: 18px;
font-weight: bold;
margin-top: 10px;
              </string>
             </property>
             <property name="text">
              <string>@username</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_1">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_dados">
           <item>
            <widget class="QLabel" name="label_secao1">
             <property name="styleSheet">
              <string notr="true">
font-size: 18px;
font-weight: bold;
margin-bottom: 10px;
              </string>
             </property>
             <property name="text">
              <string>üìù Dados Pessoais</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_nome">
             <property name="text">
              <string>Nome Completo:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nomeEdit">
             <property name="placeholderText">
              <string>Seu nome completo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_email">
             <property name="text">
              <string>Email:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emailEdit">
             <property name="placeholderText">
              <string>seu@email.com</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_curso">
             <property name="text">
              <string>Curso:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="cursoEdit">
             <property name="placeholderText">
              <string>Ex: Engenharia de Software</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_secao2">
         <property name="styleSheet">
          <string notr="true">
font-size: 18px;
font-weight: bold;
margin-top: 20px;
margin-bottom: 10px;
          </string>
         </property>
         <property name="text">
          <string>üìä Estat√≠sticas Acad√™micas</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_stats">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_total">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">font-size: 12px; color: #8E6915;</string>
              </property>
              <property name="text">
               <string>Total de Tarefas</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="totalTarefasLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 32px; font-weight: bold;</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_concluidas">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="styleSheet">
               <string notr="true">font-size: 12px; color: #8E6915;</string>
              </property>
              <property name="text">
               <string>Conclu√≠das</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="concluidasLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 32px; font-weight: bold; color: #4CAF50;</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_pendentes">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="styleSheet">
               <string notr="true">font-size: 12px; color: #8E6915;</string>
              </property>
              <property name="text">
               <string>Pendentes</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pendentesLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 32px; font-weight: bold; color: #D3AF35;</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_atrasadas">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="styleSheet">
               <string notr="true">font-size: 12px; color: #8E6915;</string>
              </property>
              <property name="text">
               <string>‚ö†Ô∏è Atrasadas</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="atrasadasLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 32px; font-weight: bold; color: #FF6B6B;</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_taxa">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="styleSheet">
               <string notr="true">font-size: 12px; color: #8E6915;</string>
              </property>
              <property name="text">
               <string>Taxa de Conclus√£o</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="taxaLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 32px; font-weight: bold;</string>
              </property>
              <property name="text">
               <string>0%</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_botoes">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="alterarSenhaButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>üîê Alterar Senha</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancelarButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>‚úï Cancelar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editarButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>Editar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="salvarButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>Salvar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
